src/Native/Unix/System.Native/pal_networking.cpp

```diff
@@ -1730,7 +1730,11 @@ extern "C" Error SystemNative_Accept(intptr_t socket, uint8_t* socketAddress, in
 
     socklen_t addrLen = static_cast<socklen_t>(*socketAddressLen);
     int accepted;
+#if defined(HAVE_ACCEPT_4) && defined(SOCK_CLOEXEC)
```

Comment:
I expect this to be present on Linux systems where .NET Core is used. It has been around for a while.
Mac doesn't have it.